ESLint-Formatter.py

```diff
@@ -143,11 +143,64 @@ def get_node_path():
     return node
 
   @staticmethod
+  def get_config_path():
+    cwd = os.getcwd()
+    folders = cwd.split('/')
+    return PluginUtils.get_eslint_config(cwd, folders)
+
+  @staticmethod
+  def get_eslint_config(root, folders):
+    cfs = ['.eslintrc.js', '.eslintrc.yaml', '.eslintrc.yml', '.eslintrc.json', '.eslintrc']
```

Comment:
I don't think we should be worrying about the configuration file. We call eslint from the command line passing the file path to be fixed. We pass a config file only if a project sets a specific one. Otherwise eslint calculates the configuration of that file based on the .eslintrc files up the tree to the root from that file.